cli/src/main/kotlin/OrtServerMain.kt

```diff
@@ -56,7 +56,7 @@ class OrtServerMain : SuspendingNoOpCliktCommand(COMMAND_NAME) {
             names = setOf("--version", "-v"),
             help = "Show the version and exit.",
             // TODO: Also add the server version given by `baseUrl` to this output.
-            message = { "$commandName CLI version $it" }
+            message = { "$commandName version $it" }
```

Comment:
Commit message: But "CLI" does not stand for "Client" 😉 So while there IMO is no "redundancy", I still agree that is reads better without "CLI".

Reply:
It does:
*  https://github.com/eclipse-apoapsis/ort-server/discussions/1794#discussioncomment-11830535
* https://github.com/eclipse-apoapsis/ort-server/blob/efadae752d97db1b23256bb6ca52e5b293b87154/cli/src/main/kotlin/OrtServerMain.kt#L64

Reply:
No 😉 "OSC" stands for "ORT Server Client" and "CLI" stands for "Command Line Interface", but neither does "CLI" stand for "client", nor does the "OSC" acronym in any way refer to "client" literally.

Reply:
I think I misunderstood your first comment, I'm using a different rationale now.